indico/modules/rb_new/client/js/common/bookings/BookingEditForm.jsx

```diff
@@ -208,35 +208,40 @@ class BookingEditForm extends React.Component {
         const today = moment();
         const bookingFinished = today.isAfter(startDt, 'day') && (endDt && today.isAfter(endDt, 'day'));
 
+        const onlySingle = !hideOptions.single && hideOptions.recurring && hideOptions.daily;
+        const onlyDaily = hideOptions.single && hideOptions.recurring && !hideOptions.daily;
+        const onlyRecurring = hideOptions.single && !hideOptions.recurring && hideOptions.daily;
```

Comment:
You could simplify these checks like this:

```js
// all but one option are hidden
const showRecurrenceOptions = ['single', 'daily', 'recurring'].filter(x => hideOptions[x]).length !== 2;
```

and then just use `showRecurrenceOptions` below